packages/eslint-plugin-sui/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "eslint-plugin-sui",
+  "version": "0.0.0",
+  "description": "Set of sui lint rules",
+  "keywords": [
+    "eslint",
+    "eslintplugin",
+    "eslint-plugin"
+  ],
+  "author": "Sui",
+  "main": "./src/index.js",
+  "exports": "./src/index.js",
+  "scripts": {
+    "lint": "npm-run-all \"lint:*\"",
+    "lint:eslint-docs": "npm-run-all \"update:eslint-docs -- --check\"",
+    "lint:js": "eslint .",
+    "test": "npx sui-test server -P ./tests/**/*.js",
+    "update:eslint-docs": "eslint-doc-generator"
+  },
+  "dependencies": {
+    "requireindex": "^1.2.0",
```

Comment:
All the external dependencies should be saved in a strict way, so caret should be removed.

Reply:
👍